plugin/pkg/admission/defaultingressclass/admission_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"reflect"
 	"testing"
 
+	networkingv1 "k8s.io/api/networking/v1"
 	networkingv1beta1 "k8s.io/api/networking/v1beta1"
```

Comment:
what in this file is still using `networkingv1beta1 "k8s.io/api/networking/v1beta1"`?